applications/mne_anonymize/mne_anonymize.pro

```diff
@@ -69,12 +69,14 @@ CONFIG(debug, debug|release) {
 DESTDIR = $${MNE_BINARY_DIR}
 
 SOURCES += main.cpp \
-    settingscontroller.cpp \
-    fiffanonymizer.cpp
+    fiffanonymizer.cpp \
+    mainwindow.cpp \
+    settingscontrollerCL.cpp
```

Comment:
`settingscontrollerCL.cpp` should be `settingscontrollercl.cpp` captial letters in file names are discouraged.

Reply:
done